rna.clj

```diff
@@ -0,0 +1,12 @@
+(ns exercises.rna)
+         
+(defn dna->rna [nucleotide]
+  (case nucleotide
+    \C \G
+    \G \C
+    \A \U
+    \T \A
+    (throw (AssertionError. "Invalid nucleotide"))))
+
+(defn to-rna [dna]
+  (apply str (map dna->rna dna)))
```

Comment:
Taking advantages of `(->>)` thread-last macro, this could become the following:

```suggestion
  (->> dna (map dna->rna) (apply str))
```

Reply:
👍